apps/web/components/apps/onboarding/ConfigureStepCard.tsx

```diff
@@ -18,13 +21,20 @@ type ConfigureStepCardProps = {
 };
 
 export const ConfigureStepCard: FC<ConfigureStepCardProps> = ({ slug, eventType, onSave, loading }) => {
+  const { t } = useLocale();
   const { getAppDataGetter, getAppDataSetter } = useAppsData();
+  const { shouldLockDisableProps } = useLockedFieldsManager(
+    eventType,
+    t("locked_fields_admin_description"),
+    t("locked_fields_member_description")
+  );
+
   const data = getAppDataGetter(slug as EventTypeAppsList)("") as Record<string, unknown>;
   return (
     <StepCard>
       <EventTypeAppSettings
         slug={slug}
-        disabled={false}
+        disabled={shouldLockDisableProps("apps").disabled}
```

Comment:
not sure if it is needed to check disabled props from `shouldLockDisableProps` for this specific usecase